src/openpersonen/api/management/commands/perform_csv_dataset_test.py

```diff
@@ -0,0 +1,119 @@
+from django.core.management import BaseCommand
+from django.db.models import Q
+from django.urls import reverse
+
+import requests
+from rest_framework.authtoken.models import Token
+
+from openpersonen.accounts.models import User
+from openpersonen.api.demo_models import *
+
+user, _ = User.objects.get_or_create(username="test")
+token, _ = Token.objects.get_or_create(user=user)
+
+
+class Command(BaseCommand):
+    """
+    Run using python src/manage.py perform_csv_dataset_test
+    """
+
+    help = "Test APIs using demo data in database.  Note: Must have server running at http://localhost:8000"
+
+    def make_request(self, viewname, viewname_kwargs):
+
+        url = reverse(
+            viewname,
+            kwargs=viewname_kwargs,
+        )
+
+        response = requests.get(
+            f"http://localhost:8000{url}",
+            headers={"Authorization": f"Token {token}"},
+        )
+
+        if response.status_code != 200:
+            print(
+                f"{url} gave a response code of {response.status_code}.  Expected 200"
+            )
+
```

Comment:
I think this would work too, to stick with the idea of this command being a testcase
```suggestion
        assert response.status_code == 200, (
            f"{url} gave a response code of {response.status_code}.  Expected 200"
        )
```

Reply:
Won't using an `AssertionError` and cause the script to stop running?  The reason I wanted to just print is so that it kept running and the user could see all the endpoints that would not return a 200.

Reply:
Ah yeah sorry, that's right. I forgot that this is one big testcase and in that case it would stop early indeed. I think this approach is okay then, though maybe you could use something like this (instead of print) for the error output:
```python
self.stdout.write(self.style.ERROR("..."))
```
That way the terminal output will be red for errors. Here's some of the other colors you could use, if you need them: https://docs.djangoproject.com/en/3.1/ref/django-admin/#syntax-coloring

Reply:
I did double check and this does raise an exception and stop the script from running which isn't what I want for the script. 

The reason for the script is that some test data files they give us can be very large (eg. the one I got was 2000 lines, thus 2000 instances).    With the script it will run over all the potential url calls that could be made and let you know which ones would not return a 200.

Reply:
I can change the name of this script if you think have 'test' makes it unclear how the script will behave :)

Reply:
@stevenbal I commented my additional two comments before I saw yours :P I guess github doesn't updated in real time.

I like your suggestion so I'll change it to use that.